pkg/accounts/service/follow.ts

```diff
@@ -0,0 +1,32 @@
+import { Result } from '@mikuroxina/mini-fn';
+
+import type { ID } from '../../id/type.js';
+import type { AccountID } from '../model/account.js';
+import { AccountFollow } from '../model/follow.js';
+import type { AccountFollowRepository } from '../model/repository.js';
+
+export class FollowService {
+  private readonly followRepository: AccountFollowRepository;
+
+  constructor(followRepository: AccountFollowRepository) {
+    this.followRepository = followRepository;
+  }
```

Comment:
```suggestion
  constructor(private readonly followRepository: AccountFollowRepository) {}
```
コンストラクタ引数をそのままメンバーにできると思います. `UnfollowService` も同様です.